bin/phpcbf

```diff
@@ -8,6 +8,6 @@
  * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
  */
 
-require_once __DIR__.'/../autoload.php';
+require_once __DIR__.'/../vendor/autoload.php';
```

Comment:
No, it works on the out-of-the box PHP_CodeSniffer. Instead of requiring code sniffer's autoload file, we delegate autoloading to Composer and thus make sure all other dependencies will be loaded as well.

If at any point in time PHP_CodeSniffer will depend on anything, it will require this change anyways.